city_scrapers/spiders/il_pollution_control.py

```diff
@@ -0,0 +1,164 @@
+import json
+from datetime import datetime
+from urllib.parse import urljoin
+
+import scrapy
+from city_scrapers_core.constants import BOARD, CANCELLED, FORUM, NOT_CLASSIFIED, PASSED, TENTATIVE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlPollutionControlSpider(CityScrapersSpider):
+    name = "il_pollution_control"
+    agency = "Illinois Pollution Control Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["pcb.illinois.gov"]
+    start_urls = ["https://pcb.illinois.gov/ClerksOffice/MeetingMinutes"]
+    json_url = "https://pcb.illinois.gov/ClerksOffice/GetCalendarEvents"
+
+    def __init__(self, *args, **kwargs):
+        self.link_map = dict()  # Populated by self._parse_minutes()
+        self.link_parse_complete = False
+        super().__init__(*args, **kwargs)
+
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        """ Overridden `from_crawler` to connect `spider_idle` signal. """
+        spider = super(IlPollutionControlSpider, cls).from_crawler(crawler, *args, **kwargs)
```

Comment:
In Python 3 you should be able to just call `super()` without specifying the class

Reply:
Good point! I'll fix that.